docs/asyncapi.yaml

```diff
@@ -470,14 +470,18 @@ components:
       minimum: 1000000000
       maximum: 9999999999
       example: 1234567890
+
     FilenameSchedule:
       type: string
-      description: The PDF filename for the payment schedule
-      example: FFC_PaymentSchedule_SFI_2022_1234567890_2023051810180298.pdf
+      description: The PDF filename for the payment schedule, formatted as product code using 3 to 6 uppercase characters, descriptor in PascalCas
+      example: FFC_PaymentSchedule_SFI_2022_1234567890_202305181018.pdf
+      pattern: ^[A-Z]{3,6}\_[A-Z]([A-Z0-9]*[a-z][a-z0-9]*[A-Z]|[a-z0-9]*[A-Z][A-Z0-9]*[a-z])[A-Za-z0-9]*\_[A-Z]{3,6}_\d{4}_\d{10}_\d{12}\.pdf$
     FilenameStatement:
       type: string
-      description: The PDF filename for the payment statement
-      example: FFC_PaymentStatement_SFI_2022_1234567890_2023051810180298.pdf
+      description: The PDF filename for the payment statement, formatted as product code using 3 to 6 uppercase characters, descriptor in PascalCa
+      example: FFC_PaymentStatement_SFI_2022_1234567890_202305181018.pdf
+      pattern: ^[A-Z]{3,6}\_[A-Z]([A-Z0-9]*[a-z][a-z0-9]*[A-Z]|[a-z0-9]*[A-Z][A-Z0-9]*[a-z])[A-Za-z0-9]*\_[A-Z]{3,6}_\d{4}_\d{10}_\d{12}\.pdf$ 
```

Comment:
Any reason for reducing the length of the time-stamp section of filename from 16 to 12?

Reply:
The spec file we received showed that the final element was to be 12 digits.